README.md

```diff
@@ -33,7 +33,8 @@ The following are the corresponding ``torchaudio`` versions and supported Python
 
 | ``torch``                | ``torchaudio``           | ``python``                      |
 | ------------------------ | ------------------------ | ------------------------------- |
-| ``master`` / ``nightly`` | ``main`` / ``nightly``   | ``>=3.7``, ``<=3.9``            |
+| ``master`` / ``nightly`` | ``main`` / ``nightly``   | ``>=3.7``, ``<=3.10``           |
+| ``1.11.0``               | ``0.11.0``               | ``>=3.7``, ``<=3.10``           |
```

Comment:
We should not mention `1.11.0 / 0.11` until at least RC is out, otherwise it is confusing for users without context.